tests/unit_tests/config/test_ert_config.py

```diff
@@ -1644,3 +1645,74 @@ def test_that_empty_params_file_gives_reasonable_error(tmpdir, param_config):
 
         with pytest.raises(ConfigValidationError, match="No parameters specified in"):
             ErtConfig.from_file("config.ert")
+
+
+@pytest.mark.usefixtures("use_tmpdir")
+@pytest.mark.parametrize(
+    "max_running_queue_config_entry",
+    [
+        pytest.param(
+            """
+        MAX_RUNNING 6
+        QUEUE_OPTION LSF MAX_RUNNING 2
+        QUEUE_OPTION SLURM MAX_RUNNING 2
+        """,
+            id="general_keyword_max_running",
+        ),
+        pytest.param(
+            """
+        QUEUE_OPTION TORQUE MAX_RUNNING 6
+        MAX_RUNNING 2
+        """,
+            id="queue_option_max_running",
+        ),
+    ],
+)
+def test_queue_config_general_max_running_takes_precedence_over_queue_option(
```

Comment:
Hm, is it really so? I thought that the queue_option takes precedence ... always